modules/sdcore-control-plane-k8s/main.tf

```diff
@@ -0,0 +1,176 @@
+resource "juju_model" "sdcore" {
+  name = var.model_name
+}
+
+module "sdcore-amf-k8s" {
+  source                 = "git::https://github.com/canonical/sdcore-amf-k8s-operator//terraform"
```

Comment:
Are the two // needed?  `sdcore-amf-k8s-operator//terraform` vs `sdcore-amf-k8s-operator/terraform`.  All the sources have that

Reply:
We need  use sdcore-amf-k8s-operator//terraform as the modules are kept under a subdirectory. Please check:  https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories

Reply:
Understood: 
> A special double-slash syntax is interpreted by Terraform